config/registry_image_pruner/cronjob.yaml

```diff
@@ -29,9 +29,7 @@ spec:
             command:
               - /bin/bash
               - '-c'
-              - >-
-                pip install -r /image-pruner/requirements.txt && python
-                /image-pruner/prune_images.py --namespace $(NAMESPACE)
+              - python /image-pruner/prune_images.py --namespace $(NAMESPACE)
```

Comment:
Hmm, how does this work? `$(NAMESPACE)` doesn't look like the correct syntax for a bash env var :thinking:

Reply:
that's a kubernetes thingy https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#using-environment-variables-inside-of-your-config

Reply:
nice, TIL